usaspending_api/common/threaded_data_loader.py

```diff
@@ -34,10 +36,13 @@ class ThreadedDataLoader():
     #   pre_row_function - Like post_row_function, but before the model class is updated
     #   post_process_function - A function to call when all rows have been processed, uses the same
     #                           function parameters as post_row_function
-    def __init__(self, model_class, threads=10, field_map={}, value_map={}, collision_field=None, collision_behavior='update', pre_row_function=None, post_row_function=None, post_process_function=None):
+    def __init__(self, model_class, processes=-1, field_map={}, value_map={}, collision_field=None, collision_behavior='update', pre_row_function=None, post_row_function=None, post_process_function=None):
```

Comment:
Small detail, but would it make sense to use `None` instead? Magic numbers are always confusing.

Also, unrelated to this PR, but using objects like `{}` and `[]` as default parameters leads to some unexpected problems. Those are initialized _once_ (at function creation time, not call time), which means they are shared across function calls.